packages/autorest.typescript/src/autorestSession.ts

```diff
@@ -44,6 +44,7 @@ export interface AutorestOptions {
   lenientModelDeduplication?: boolean;
   useLegacyLro?: boolean;
   flavor?: PackageFlavor;
+  moduleKind?: "cjs" | "esm";
```

Comment:
Alternatively this can be `esm?: boolean` since we can only have esm or cjs and cjs is the default

Reply:
Decided to stick to moduleKind to match existing verbiage in rlc-comment and typespec-ts